src/analysis/outline.ml

```diff
@@ -49,21 +49,33 @@ let get_class_field_desc_infos = function
   | _ -> None
 
 let rec summarize node =
+  let verbosity = 0 in
   let location = node.t_loc in
   match node.t_node with
-  | Value_binding vb      -> id_of_patt vb.vb_pat >>| mk `Value ~location
-  | Value_description vd  -> Some (mk `Value ~location vd.val_id)
+  | Value_binding vb      ->
+    begin match id_of_patt vb.vb_pat with
+    | None -> None
+    | Some ident ->
+      let outline_type =
+        let ppf, to_string = Format.to_string () in
+        Type_utils.print_type_with_decl ~verbosity node.t_env ppf vb.vb_pat.pat_type;
```

Comment:
If I'm reading the diff correctly, this is the only use of `verbosity`. If that's the case, could you inline it its definition here and remove the binding? I think it'd make things more readable.

Reply:
Fixed this1